examples/ofac-search-ui/src/setupProxy.js

```diff
@@ -0,0 +1,12 @@
+const proxy = require("http-proxy-middleware");
+
```

Comment:
I figure this proxy setup is useful for local dev? With the `/api/` prefix dropped do we need to handle that specifically for local dev or could it also serve off of `/`?